pages/news.html

```diff
@@ -161,7 +161,8 @@ <h1>NEWS</h1>
             <p><strong>14/09/2018</strong> - <a href="https://lists.opensuse.org/uyuni-devel" target="_blank">uyuni-devel</a> mailing list is now open for everyone, for development discussions.</p>
             <p><strong>21/08/2018</strong> - <a href="https://twitter.com/kkaempf" target="_blank">Klaus Kämpf</a> will be discussing about Uyuni at talk <a href="https://cfp.all-systems-go.io/en/ASGpublic/events/182" target="_blank">From Physical to Cloud to Container</a>, next 2018-09-28 at <a href="https://all-systems-go.io/" target="_blank">Schedule All Systems Go! 2018</a>.</p>
             <p><strong>20/08/2018</strong> - Uyuni source code available at GitHub <a href="https://github.com/uyuni-project/uyuni" target="_blank">uyuni-project/uyuni</a>.
-            <p><strong>29/05/2018</strong> - Official #uyuni IRC channel at <a href="http://freenode.net" target="_blank">Freenode</a>. Join with your favourite IRC client, or use the <a href="http://webchat.freenode.net/?channels=%23uyuni" target="_blank">web application</a>.</p>
+            <!--
+             <p><strong>29/05/2018</strong> - Official #uyuni IRC channel at <a href="http://freenode.net" target="_blank">Freenode</a>. Join with your favourite IRC client, or use the <a href="http://webchat.freenode.net/?channels=%23uyuni" target="_blank">web application</a>.</p>-->
```

Comment:
Leave it uncommented, for historic reasons. This is a list of news, in the end, and there's a new entry now that says we moved to Gitter.